src/NzbDrone.Common/Instrumentation/Sentry/SentryCleanser.cs

```diff
@@ -11,26 +11,41 @@ public static SentryEvent CleanseEvent(SentryEvent sentryEvent)
         {
             try
             {
-                sentryEvent.Message = CleanseLogMessage.Cleanse(sentryEvent.Message);
+                if (sentryEvent.Message is not null)
+                {
+                    sentryEvent.Message.Formatted = CleanseLogMessage.Cleanse(sentryEvent.Message.Formatted);
+                    sentryEvent.Message.Message = CleanseLogMessage.Cleanse(sentryEvent.Message.Message);
+                    sentryEvent.Message.Params = sentryEvent.Message.Params?.Select(x => CleanseLogMessage.Cleanse(x switch
+                    {
+                        string str => str,
+                        _ => x.ToString()
+                    })).ToList();
+                }
 
-                if (sentryEvent.Fingerprint != null)
+                if (sentryEvent.Fingerprint.Any())
```

Comment:
It would throw but this field can't be null. The Sentry SDK has nullability enabled so the IDE was giving me a warning. 

https://github.com/getsentry/sentry-dotnet/blob/f9569d06eb3ba82ccdfcd0df81cfd96e3067bd10/src/Sentry/SentryEvent.cs#L153

Reply:
> but this field can't be null

Perfect, this is all I was ensuring. I assume `sentryEvent.Extra` is similar?

Reply:
Sure thing, just a few lines away at: https://github.com/getsentry/sentry-dotnet/blob/f9569d06eb3ba82ccdfcd0df81cfd96e3067bd10/src/Sentry/SentryEvent.cs#L166